test/src/unit-capi-array.cc

```diff
@@ -106,7 +106,6 @@ struct ArrayFx {
   void create_dense_vector(const std::string& path);
   void create_dense_array(const std::string& path);
   void write_fragment(tiledb_array_t* array, uint64_t timestamp);
-  static std::string random_name(const std::string& prefix);
```

Comment:
There was really an impressive amount of combo copypasta+deadcode here, wasn't there.